src/include/log.h

```diff
@@ -13,32 +34,43 @@
 /* Logging subsystem declarations. */
 #define	WT_LOG_ALIGN			128
 
+/*
+ * Atomically set the two components of the LSN.
+ */
+#define	lsn_file		u.l.file
+#define	lsn_offset		u.l.offset
+#define	WT_SET_LSN(l, f, o) (l)->u.file_offset = (((uint64_t)(f) << 32) + (o))
```

Comment:
Ack, just trying to avoid mixing bit shifts with union field accesses.  We could do it here (by assigning to the fields of a local variable then using its `file_offset`) but it's not worth it.